templates/event-translations-footer.php

```diff
@@ -33,4 +33,4 @@ function set_translation_table_<?php echo esc_html( $translation_set_id ); ?>()
 </script>
 <?php
 gp_enqueue_script( 'wporg-translate-editor' );
-gp_tmpl_footer(); ?>
```

Comment:
I'm not sure I understand why this fixes the issue, does this mean that `templates/event-translations-footer.php` is being included multiple times? I searched for `event-translations-footer.php` but it only seems to be included here, so doesn't appear to be included multiple times.

Reply:
You are correct, just realised I hadn't moved it outside the for loop. Will continue investigating to determine the true cause or why this fixes it, thank you!